examples/UberSDK/UberSDK/ContentView.swift

```diff
@@ -61,11 +61,14 @@ final class Content {
                 prefill: isPrefillExpanded ? prefillBuilder.prefill : nil
             ),
             completion: { result in
-                switch result {
-                case .success(let client):
-                    self.response = "\(client)"
-                case .failure(let error):
-                    self.response = error.localizedDescription
+                // Slight delay to allow for ASWebAuthenticationSession dismissal
+                DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
```

Comment:
could we use a callback instead?

Reply:
We are using the callback on the ASWeb side, unfortunately this doesn't seem to account for the animation. This will only be an issue for the sample app most likely.